### PR TITLE
TW 345 Billing notifications

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

File: src/pages/docs/administration/billing.md [hunk 1]
```diff
@@ -186,13 +182,11 @@ Add or update your company name, billing email, address, and your VAT ID if appl
...
-
-<img alt="Failed payment email" src="https://assets.postman.com/postman-docs/billing-email-payment-declined-v9.1.jpg" width="400px"/>
+If you have a payment method on file, Postman will automatically attempt three times to charge it. If this is unsuccessful, [Billing team members](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) receive an email notification. Use the link in the email to access and pay for your [invoice](#using-the-pay-now-option).
 
 If payment is unsuccessful, Postman offers a grace period of 21 days to avoid service disruption. For invoice-based teams, this means 21 days following the due date listed on the invoice.
 
-> All team members receive a notification in the event of non-payment and service deactivation in Postman.
+In the event of non-payment and service deactivation, Postman sends an email notification to Billing team members.
```
Comment (taehoshino) [inline src/pages/docs/administration/billing.md:189]: We send email notification to the billing email address in billing details page in(http://go.postman.co/settings/team/billing-details).

File: src/pages/docs/administration/billing.md [hunk 2]
```diff
@@ -204,7 +198,7 @@ To change your plan right away, open your [billing dashboard](https://go.postman
 
 <img alt="Edit plan page" src="https://assets.postman.com/postman-docs/billing-edit-plan-v9.1.jpg"/>
 
-You can opt to apply changes right away or on renewal. You can adjust your team size, billing frequency, and plan type. Select **Next** to adjust your team's add-ons, including monitoring blocks, mock server blocks, and custom domains. Select **Next** to then **Confirm Changes**.
+You can opt to apply changes right away or on renewal. You can adjust your team size, billing frequency, and plan type. Select **Next** to adjust your team's add-ons, including monitoring blocks, mock server blocks, and custom domains. Select **Next** to then **Confirm Changes**. Postman will send an email notification about the plan change to Billing team members.
```
Comment (taehoshino) [inline src/pages/docs/administration/billing.md:201]: Again, not to Billing team members, but to the billing email address in billing details page.

File: src/pages/docs/administration/billing.md [hunk 3]
```diff
@@ -218,7 +212,7 @@ The cost of your new plan and any new seats, if applicable, are prorated based o
 
 Monthly teams can switch to an annual plan at any time by navigating to the [billing dashboard](https://go.postman.co/billing/overview) and selecting **Edit Plan** on the right.
 
-Select to pay **Monthly** or **Annual** and confirm your changes. Note that annual teams can only switch to a monthly plan at the end of their current billing cycle. You can find the end date of your current billing cycle under **Overview**.
+Select to pay **Monthly** or **Annual** and confirm your changes. Note that annual teams can only switch to a monthly plan at the end of their current billing cycle. You can find the end date of your current billing cycle under **Overview**. Postman will send an email notification about the billing cycle change to Billing team members.
```
Comment (taehoshino) [inline src/pages/docs/administration/billing.md:215]: We send email notification to billing email address in billing details page.

File: src/pages/docs/administration/billing.md [hunk 4]
```diff
@@ -236,21 +230,7 @@ Update the number of monitoring blocks, mock server blocks, and custom domains y
...
-<img alt="Auto-Flex opt in early notification" src="https://assets.postman.com/postman-docs/auto-flex-opt-in-notification-v9.1.jpg" />
-
-Select **Get Early Access**.
-
-<img alt="Auto-Flex opt in confirmation" src="https://assets.postman.com/postman-docs/opt-in-to-auto-flex-modal.jpg" width="400px"/>
-
-Select to **Agree to Auto-Flex's terms and conditions**, then select **Get Early Access**.
-
-> View [Postman's Auto-Flex team policy](https://learning.postman.com/auto-flex-policy/).
+Auto-Flex is automatically enabled for all Postman Free, Basic, and Professional plans.
```
Comment (taehoshino) [inline src/pages/docs/administration/billing.md:233]: Free teams don't have auto-flex. cc: @Kalpana-Postman

**Review by taehoshino — COMMENTED** [review]
Thanks for the changes! Mostly looks good. Please see my comments.

**Comment by HeatherStenson** [comment]
Thank you @taehoshino! Updated per your comments.

**Review by claireriley — APPROVED** [review]
LGTM! ⭐

**Pull request merged**